TESSDATA_PREFIX is interpreted incorrectly

I'm trying to get the https://github.com/bertsky/workflow-configuration/blob/master/crop-anyocr-binarize-page-olena-sauvola-denoise-ocropy-deskew-page-ocropy-segment-tesseract-ocropy-dewarp-ocr-ocropy-tesseract.mk running on https://github.com/OCR-D/assets/tree/master/data/kant_aufklaerung_1784/data

```sh
ocrd-make -f gt-binarize-page-olena-sauvola-denoise-ocropy-deskew-page-ocropy-clip-deskew-region-tesseract-resegment-dewarp-ocr-ocropy-tesseract-extract-lines.mk INPUT
=OCR-D-IMG LOGLEVEL=DEBUG
```


If I set `TESSDATA_PREFIX` like it is supposed to, to the parent directory of `tessdata`

```
19:38:21.064 DEBUG processor.TesserocrRecognize - TESSDATA: /home/kba/ocrd_all/venv/share/, installed Tesseract models: ['tessdata/Fraktur_50000000.334_450937-best', 'tess
data/Fraktur_50000000.334_450937-fast', 'tessdata/best/Fraktur_50000000.334_450937', 'tessdata/eng', 'tessdata/equ', 'tessdata/fast/Fraktur_50000000.334_450937', 'tessdata
/osd', 'tessdata/script/Fraktur']
Traceback (most recent call last):
  File "/home/kba/ocrd_all/venv/bin/ocrd-tesserocr-recognize", line 8, in <module>
    sys.exit(ocrd_tesserocr_recognize())
  File "/home/kba/ocrd_all/venv/lib/python3.6/site-packages/click/core.py", line 829, in __call__
    return self.main(*args, **kwargs)
  File "/home/kba/ocrd_all/venv/lib/python3.6/site-packages/click/core.py", line 782, in main
    rv = self.invoke(ctx)
  File "/home/kba/ocrd_all/venv/lib/python3.6/site-packages/click/core.py", line 1066, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/home/kba/ocrd_all/venv/lib/python3.6/site-packages/click/core.py", line 610, in invoke
    return callback(*args, **kwargs)
  File "/home/kba/ocrd_all/venv/lib/python3.6/site-packages/ocrd_tesserocr/cli.py", line 36, in ocrd_tesserocr_recognize
    return ocrd_cli_wrap_processor(TesserocrRecognize, *args, **kwargs)
  File "/home/kba/monorepo/core/ocrd/ocrd/decorators.py", line 54, in ocrd_cli_wrap_processor
    run_processor(processorClass, ocrd_tool, mets, workspace=workspace, **kwargs)
  File "/home/kba/monorepo/core/ocrd/ocrd/processor/base.py", line 60, in run_processor
    processor.process()
  File "/home/kba/ocrd_all/venv/lib/python3.6/site-packages/ocrd_tesserocr/recognize.py", line 87, in process
    raise Exception("configured model " + sub_model + " is not installed")
Exception: configured model script/Fraktur is not installed
```

If I set `TESSDATA_PREFIX=$TESSDATA_PREFIX/tessdata`, it still cannot find the model:

```
19:41:33.510 DEBUG processor.TesserocrRecognize - TESSDATA: /home/kba/ocrd_all/venv/share/tessdata/, installed Tesseract models: ['Fraktur_50000000.334_450937-best', 'Frak
tur_50000000.334_450937-fast', 'best/Fraktur_50000000.334_450937', 'eng', 'equ', 'fast/Fraktur_50000000.334_450937', 'osd', 'script/Fraktur']
Traceback (most recent call last):
  File "/home/kba/ocrd_all/venv/bin/ocrd-tesserocr-recognize", line 8, in <module>
    sys.exit(ocrd_tesserocr_recognize())
  File "/home/kba/ocrd_all/venv/lib/python3.6/site-packages/click/core.py", line 829, in __call__
    return self.main(*args, **kwargs)
  File "/home/kba/ocrd_all/venv/lib/python3.6/site-packages/click/core.py", line 782, in main
    rv = self.invoke(ctx)
  File "/home/kba/ocrd_all/venv/lib/python3.6/site-packages/click/core.py", line 1066, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/home/kba/ocrd_all/venv/lib/python3.6/site-packages/click/core.py", line 610, in invoke
    return callback(*args, **kwargs)
  File "/home/kba/ocrd_all/venv/lib/python3.6/site-packages/ocrd_tesserocr/cli.py", line 36, in ocrd_tesserocr_recognize
    return ocrd_cli_wrap_processor(TesserocrRecognize, *args, **kwargs)
  File "/home/kba/monorepo/core/ocrd/ocrd/decorators.py", line 54, in ocrd_cli_wrap_processor
    run_processor(processorClass, ocrd_tool, mets, workspace=workspace, **kwargs)
  File "/home/kba/monorepo/core/ocrd/ocrd/processor/base.py", line 60, in run_processor
    processor.process()
  File "/home/kba/ocrd_all/venv/lib/python3.6/site-packages/ocrd_tesserocr/recognize.py", line 87, in process
    raise Exception("configured model " + sub_model + " is not installed")
Exception: configured model script/Latin is not installed
Makefile:307: recipe for target 'OCR-D-OCR-TESS-Fraktur+Latin-BINPAGE-sauvola-DENOISE-ocropy-DESKEW-tesseract-DESKEW-ocropy-CLIP-DESKEW-tesseract-DESKEW-ocropy-RESEG-DEWAR
P' failed
```

Has somebody an idea what I am doing wrong?

--- a/CHANGELOG.md
+++ b/CHANGELOG.md
@@ -11,7 +11,7 @@ Changed:
 
 Fixed:
 
-  * Debug statements listed the wrong model paths, #129
+  * Always set path to `TESSDATA_PREFIX` for `tesserocr.get_languages`, #129
 
 ## [0.8.3] - 2020-05-12
 

--- a/CHANGELOG.md
+++ b/CHANGELOG.md
@@ -9,6 +9,10 @@ Changed:
 
   * segment-region: in `sparse_text` mode, also add text lines
 
+Fixed:
+
+  * Debug statements listed the wrong model paths, #129
+
 ## [0.8.3] - 2020-05-12
 
 Fixed:

--- a/Dockerfile
+++ b/Dockerfile
@@ -5,6 +5,7 @@ ENV PYTHONIOENCODING utf8
 
 WORKDIR /build-ocrd
 COPY setup.py .
+COPY ocrd_tesserocr/ocrd-tool.json .
 COPY README.md .
 COPY requirements.txt .
 COPY requirements_test.txt .

--- a/ocrd_tesserocr/recognize.py
+++ b/ocrd_tesserocr/recognize.py
@@ -5,7 +5,7 @@ from PIL import Image, ImageStat
 
 from tesserocr import (
     RIL, PSM,
-    PyTessBaseAPI, get_languages)
+    PyTessBaseAPI, get_languages as get_languages_)
 
 from ocrd_utils import (
     getLogger,
@@ -45,6 +45,12 @@ LOG = getLogger('processor.TesserocrRecognize')
 
 CHOICE_THRESHOLD_NUM = 6 # maximum number of choices to query and annotate
 CHOICE_THRESHOLD_CONF = 0.2 # maximum score drop from best choice to query and annotate
+
+def get_languages(*args, **kwargs):
+    """
+    Wraps tesserocr.get_languages() with a fixed path parameter.
+    """
+    return get_languages_(*args, path=TESSDATA_PREFIX, **kwargs)
 
 class TesserocrRecognize(Processor):
 

